packages/eslint-visitor-keys/package.json

```diff
@@ -28,26 +28,24 @@
   "devDependencies": {
     "@types/estree": "^0.0.51",
     "@types/estree-jsx": "^0.0.1",
-    "@typescript-eslint/parser": "^5.14.0",
+    "@typescript-eslint/parser": "^8.7.0",
     "c8": "^7.11.0",
     "chai": "^4.3.6",
     "eslint-release": "^3.2.0",
     "esquery": "^1.4.0",
     "json-diff": "^0.7.3",
     "mocha": "^9.2.1",
     "opener": "^1.5.2",
-    "rollup": "^2.70.0",
-    "rollup-plugin-dts": "^4.2.3",
-    "tsd": "^0.19.1",
-    "typescript": "^4.9.5"
+    "rollup": "^4.22.4",
+    "rollup-plugin-dts": "^6.1.1",
+    "tsd": "^0.31.2",
+    "typescript": "^5.6.2"
   },
   "scripts": {
     "build": "npm run build:cjs && npm run build:types",
     "build:cjs": "rollup -c",
     "build:debug": "npm run build:cjs -- -m && npm run build:types",
-    "build:keys": "node tools/build-keys-from-ts",
-    "build:types": "tsc",
-    "prepare": "npm run build",
```

Comment:
If we just remove the `prepare` script, I believe `npm run build` won't be run during releases.

Reply:
We already run `build` during releases:
https://github.com/eslint/js/blob/dd7fcbbbe72b834999c9b490b8278cc1aa7209d7/.github/workflows/release-please.yml#L39-L41